capstone/src/main/resources/static/event.js

```diff
@@ -56,7 +56,7 @@ function createEventElement(eventListElement, event, displaySaveButton) {
 
   // Displays only for individual users
   if (displaySaveButton) {
-    eventElement.appendChild(createSaveEventButton(event));
```

Comment:
Does this change belong in this PR? Looks like something that might have dropped in from another one.

Reply:
I have no idea where this came from. I'm reverting the change